portfolio/src/main/webapp/index.html

```diff
@@ -23,20 +24,32 @@ <h1>Software Engineer and Cybersecurity Student</h1>
     </div>
 
     <!-- Navigation -->
-    <nav class="z-depth-2">
-      <p>Future links to other pages will go here.</p>
+    <nav>
+      <div class="nav-wrapper container">
+        <a 
+          href="#" 
+          class="brand-logo">
+          Christian Liguori
+        </a>
+        <ul 
+          id="nav-mobile" 
+          class="right hide-on-med-and-down">
+          <li><a href="#about">About</a></li>
+          <li><a href="#projects">Projects</a></li>
+          <li><a href="https://github.com/cl2112">Github</a></li>
+          <li><a href="https://www.linkedin.com/in/christian-liguori">LinkedIn</a></li>
+        </ul>
+      </div>
     </nav>
     
     <!-- Biography -->
     <div id="bio-container">
-      <img 
-        src="images/Zen-5.jpg" 
-        alt="Image of a Zen Garden." 
-        class="bio-image responsive-img">
+      <div class="background-img-container"></div>
+      <a name="about"></a>
```

Comment:
The `name` attribute is deprecated (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attr-name). Use `id` instead (and I think you should be able to attach the `id` attribute to one of the `div`s below and remove this `<a>`).